boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,161 @@
+
+Overview
+********
+
+The LPCXpresso54114 board has been developed by NXP to enable evaluation
+of and prototyping with the LPC54110 family of MCUs and its low power
+features make it as easy as possible to get started with your project.
+LPCXpresso™ is a low-cost development platform available from NXP supporting
+NXP's ARM-based microcontrollers.LPCXpresso is an end-to-end solution enabling 
+embedded engineers to develop their applications from initial evaluation to
+final production.
+
+.. image:: lpcxpresso54114.png
+   :width: 720px
+   :align: center
+   :alt: LPCXPRESSO54114
+
+Hardware
+********
+
+- LPC54114 dual-core (M4F and dual M0) MCU running at up to 100 MHz
+- On-board high-speed USB based debug probe with CMSIS-DAP and J-Link protocol 
+  support, can debug the on-board LPC54114 or an external target
+- External debug probe option
+- Tri-color LED, target Reset, ISP & interrupt/user buttons for easy testing 
+  of software functionality
+- Expansion options based on Arduino UNO and Pmod™, plus additional expansion 
+  port pins
+- On-board 1.8 V and 3.3 V regulators plus external power supply option
+- 8 Mb Macronix MX25R SPI flash
+- Built-in MCU power consumption and supply voltage measurement
+- UART, I²C and SPI port bridging from LPC54114 target to USB via the on-board 
+  debug probe
+- FTDI UART connector
+
+For more information about the LPC54114 SoC and LPCXPRESSO54114 board:
+
+- `LPC54114 SoC Website`_
+- `LPC54114 Datasheet`_
+- `LPC54114 Reference Manual`_
+- `LPCXPRESSO54114 Website`_
+- `LPCXPRESSO54114 User Guide`_
+- `LPCXPRESSO54114 Schematics`_
+
+Supported Features
+==================
+
+The lpcxpresso54114 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/lpcxpresso54114/lpcxpresso54114_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC54114 SoC has IOCON registers which can be used to configure the
+functionality of a pin. Following pins are configured to support the
```

Comment:
Change to:

     The following pins are configured to support the
     :ref:`blinky-sample` and :ref:`hello_world` sample applications.